File: scripts/commands/generic/addFriend.py
from intents.chat import ChatAvatarRequestIntent
from intents.chat.ChatAvatarRequestIntent.RequestType import FRIEND_ADD_TARGET

def execute(galacticManager, player, target, args):
	ghost = player.getPlayerObject()
	if ghost is None or args is None:
		return

	name = str(args.split(" ")[0]).lower()

	if not name:
		return

	ChatAvatarRequestIntent(player, name, FRIEND_ADD_TARGET).broadcast()
	return

File: scripts/commands/generic/addFriend.js
var execute = function(galManager, player, target, args) {
	var ChatAvatarRequestIntent = Java.type("intents.chat.ChatAvatarRequestIntent");
	var RequestType = Java.type("intents.chat.ChatAvatarRequestIntent.RequestType");
	var name;
	
	if(args == null) {
		return;
	}
	
	name = args.split(" ")[0].toLowerCase(java.util.Locale.English);
	
	if(name == null) {
		return;
	}
	
	new ChatAvatarRequestIntent(player, name, RequestType.FRIEND_ADD_TARGET).broadcast();
};